app/views/participant_signup/_contact.html.erb

```diff
@@ -0,0 +1,25 @@
+<%= f.inputs :name => "Contact Information" do -%>
```

Comment:
I see what you're saying, that makes sense. 



> On Mar 9, 2016, at 10:17 PM, Aayush Saxena <notifications@github.com> wrote:
> 
> In app/views/participant_signup/_contact.html.erb:
> 
> > @@ -0,0 +1,25 @@
> > +<%= f.inputs :name => "Contact Information" do -%>
> That was my original idea and I did that before in the previous pull request. But, I realized I had to make a lot of changes in the partials, which became redundant. For example, changing the hints, the wordings of some questions in the form etc.
> Also, I am making a few fields 'required' here which sometimes students don't fill them out/forget to fill them out in the paper intake surveys. So, this might give us better data about the students.
> 
> —
> Reply to this email directly or view it on GitHub.
>